charts/wsj2023-access/values/staging.yaml

```
@@ -1,3 +1,4 @@
 ---
 domains:
   - wsj2023-access.staging.webservices.scouterna.net
+oauth-plugin: office-oauth-wsj2023-access
```

For now there's only an app registration for staging, named like this:

```suggestion
oauth-plugin: office-oauth-wsj2023-access-staging
```